packages/fxa-graphql-api/src/gql/account.resolver.ts

```diff
@@ -567,6 +568,20 @@ export class AccountResolver {
     return { exists: false };
   }
 
+  @Mutation((returns) => BasicPayload, {
+    description: 'Consume an unblock code',
+  })
+  @CatchGatewayError
+  public async rejectUnblockCode(
```

Comment:
This is a replacement for this [route](https://github.com/mozilla/fxa/blob/d970f293d3b237bfb18e2611d1f8d5fbe0766068/packages/fxa-auth-server/lib/routes/unblock-codes.js#L102)? If so, probably update teh description above.